File: python/examples/example_expectedarrivals.py
from vesselfinder_api import VesselFinderApi
from vesselfinder_api.exceptions import ApiErrorException

v = VesselFinderApi(userkey='-- Input your userkey here --', errormode=False, save_last_info=True)

try:
    print(v.expected_arrivals(interval=10080, limit=5, locode="GIGIB"))
    print(v.get_last_info())
except ApiErrorException as e:
    print(e)

File: php/examples/example_expectedarrivals.php
<?php
include __DIR__ . '/../VesselFinderApi.php';

use API\VesselFinderApi;

$client = new VesselFinderApi('-- Input your userkey here --');

try {
    $expected_arrivals = $client->expectedArrivals("GIGIB", ['interval' => 10080, 'limit' => 5]);
    print_r($expected_arrivals);
} catch (\Exceptions\ApiErrorException $e) {
    print_r($e->getMessage() . PHP_EOL);
}